clouddriver-eureka/clouddriver-eureka.gradle

```diff
@@ -0,0 +1,3 @@
+dependencies {
```

Comment:
not sure what the advantage of having consul being in here would be. If anything, would expect different libraries to be self-contained so the dependency doesn't exist amongst them.